specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-10-03-preview/HybridCompute.json

```diff
@@ -2789,11 +3101,37 @@
           "x-ms-client-flatten": true,
           "type": "object",
           "properties": {
+            "softwareAssurance": {
+              "x-ms-client-flatten": true,
+              "type": "object",
+              "properties": {
+                "softwareAssuranceCustomer": {
+                  "type": "boolean",
+                  "description": "Specifies if this machine is licensed as part of a Software Assurance agreement."
+                }
+              }
+            },
             "esuProfile": {
               "x-ms-client-flatten": true,
               "$ref": "#/definitions/LicenseProfileArmEsuProperties",
               "description": "Hybrid Compute ESU Profile properties"
             },
+            "products": {
```

Comment:
this arrangement, having a "products" property alongside esuProfile, and then having the windowsServerProfile and iotEnterpriseProfile under that, still feels odd to me. My inclination would be to put them at the same level as esuProfile. just feels like this could cause confusion. or maybe i'm overthinking it.

Reply:
Reduced the layers. Now there's a **osProfile** below esuProfile.